README.md

```diff
@@ -11,22 +11,22 @@ docker run -it linuxbrew/alpine
 
 ## CentOS 5
 ```sh
-docker build -t linuxbrew-centos5 https://raw.githubusercontent.com/Linuxbrew/docker/master/centos5/Dockerfile
+docker build -t linuxbrew/centos/5 https://raw.githubusercontent.com/Linuxbrew/docker/master/centos/5/Dockerfile
 ```
 
 ## CentOS 6
 ```sh
-docker build -t linuxbrew-centos6 https://raw.githubusercontent.com/Linuxbrew/docker/master/centos6/Dockerfile
+docker build -t linuxbrew/centos/6 https://raw.githubusercontent.com/Linuxbrew/docker/master/centos/6/Dockerfile
```

Comment:
Is `linuxbrew/centos/6` a legal tag name? I've never seen more than one `/` separating the user and image name. I'd suggest `homebrew/brew-centos6`.

Reply:
It's perfectly valid (and makes it easier to find imho)

Reply:
News to me. It's not really typical though. I've never seen it before. I'd stick to non-slash characters in Docker images.